decidim-accountability/app/controllers/decidim/accountability/admin/projects_import_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Accountability
+    module Admin
+      # This controller allows an admin to import results from a csv file for the Accountability component
+      class ProjectsImportController < Admin::ApplicationController
+        def new
+          enforce_permission_to :create, :projects_import
```

Comment:
Can you rename the permission action to `:import_projects` to be consistent with the similar functionality in budgets.

Please also apply this change to the permissions class and below to the `create` action.